signingscript/docker.d/init_worker.sh

```diff
@@ -206,6 +206,8 @@ case $ENV in
         test_var_set 'GPG_PUBKEY_PATH'
         test_var_set 'AUTOGRAPH_REFERENCE_BROWSER_PASSWORD'
         test_var_set 'AUTOGRAPH_REFERENCE_BROWSER_USERNAME'
+        test_var_set 'AUTOGRAPH_STAGE_REFERENCE_BROWSER_PASSWORD'
```

Comment:
I'm adding new AUTOGRAPH_STAGE vars here, in mobile / PROD. We would normally access autograph stage from a staging environment...but I don't think we have a staging-reference-browser. Will this be OK?

Reply:
[staging-reference-browser](https://github.com/mozilla-releng/staging-reference-browser) exists 😃 I don't remember when it was used the last time, though.